src/SenseNet.Search.Lucene29/SenseNet.Search.Lucene29.csproj

```diff
@@ -29,4 +29,8 @@
     <PackageReference Include="SenseNet.Search.Lucene29.Lucene.Net" Version="2.9.4.301" />
   </ItemGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\Contrib.Regex\Contrib.Regex.csproj" />
```

Comment:
Make sure that the new Contrib library is either a separate package or the dll is added to this one.